.ci/mellanox/ompi_test_docker_launcher.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -eE
+
+if [ "$DEBUG" = "true" ]; then
+    set -x
+fi
+
+# WORKSPACE - root folder for OpenMPI source files
+if [ -z "${WORKSPACE}" ]
+then
+    echo "ERROR: WORKSPACE is not defined"
+    exit 1
+fi
+
+if [ -z "${OMPI_CI_DOCKER_IMAGE_NAME}" ]
+then
+    echo "ERROR: OMPI_CI_DOCKER_IMAGE_NAME is not defined"
+    exit 1
+fi
+
+# Run OMPI CI scenarios (build and test)
+docker run \
+    -v /hpc/local:/hpc/local \
+    -v /opt:/opt \
+    --network=host \
+    --uts=host \
+    --ipc=host \
+    --ulimit stack=67108864 \
+    --ulimit memlock=-1 \
+    --security-opt seccomp=unconfined \
+    --cap-add=SYS_ADMIN \
+    --device=/dev/infiniband/ \
+    --env WORKSPACE="${WORKSPACE}" \
+    --env DEBUG="${DEBUG}" \
+    --env JENKINS_RUN_TESTS=yes \
+    --user swx-jenkins \
```

Comment:
Is it required to use `swx-jenkins`?

Reply:
`swx-jenkins` is needed mainly to avoid file permissions issues in case of NFS share usage inside a docker container.

Reply:
Removed this code - AzureCI takes care on this automatically.